Sources/SpeziContact/Models/ContactOption.swift

```diff
@@ -11,6 +11,43 @@ import SwiftUI
 
 
 /// Customizable way to get in contact with an individual and usually connected to a ``Contact``.
+///
+/// ### Usage
+///
+/// The following example demonstrates the usage of the ``ContactOption`` within a ``Contact``.
+/// ```swift
+/// Contact(
+///     name: PersonNameComponents(
```

Comment:
Don't repeat the entire initializer of the `Contact` here but rather refer to the documentation of `Contact`.